csrc/moe/moe_align_sum_kernels.cu

```diff
@@ -233,15 +233,17 @@ void moe_align_block_size(torch::Tensor topk_ids, int64_t num_experts,
       (num_experts + 1) * sizeof(int32_t);
 
   bool use_global_memory = false;
```

Comment:
another way to write is that, we set use_global_memory = true.

Only set the values when some optimization can be on. So we can get rid of the else branch. :-)